pkg/descheduler/strategies/lownodeutilization.go

```diff
@@ -95,8 +98,20 @@ func LowNodeUtilization(ctx context.Context, client clientset.Interface, strateg
 		targetThresholds[v1.ResourceMemory] = MaxResourcePercentage
 	}
 
-	npm := createNodePodsMap(ctx, client, nodes)
-	lowNodes, targetNodes := classifyNodes(npm, thresholds, targetThresholds)
+	lowNodes, targetNodes := classifyNodes(
+		getNodeUsage(ctx, client, nodes, thresholds, targetThresholds),
+		// The node has to be schedulable (to be able to move workload there)
+		func(node *v1.Node, usage NodeUsage) bool {
+			if nodeutil.IsNodeUnschedulable(node) {
+				klog.V(2).Infof("Node %#v is unschedulable, thus not considered as underutilized", node.Name)
```

Comment:
@ingvagabund can you change this log statement to use `InfoS` instead of `Infof`?

Reply:
I noticed this block of commented code here:
https://github.com/kubernetes-sigs/descheduler/blob/08b2dffa4224768e07bc568adeb43704de5473d5/pkg/descheduler/node/node.go#L90-L94

I did not research any further to see where the `IsReady` function is called from.

Reply:
Through https://github.com/kubernetes-sigs/descheduler/blob/08b2dffa4224768e07bc568adeb43704de5473d5/pkg/descheduler/descheduler.go#L98. Though, that code is supposed to be copy paste from the main and changed only when really necessary.

Reply:
Done